data-model/pom.xml

```diff
@@ -224,10 +216,6 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
```

Comment:
Removed because we don't exclude `bcmail` from `tika-parsers-standard-package` anymore so we don't need to re-declare this anymore IMO.